pkg/local_object_storage/writecache/flush.go

```diff
@@ -16,12 +17,14 @@ import (
 )
 
 const (
-	// defaultFlushInterval is default time interval between successive flushes.
-	defaultFlushInterval = 10 * time.Second
 	// defaultErrorDelay is default time delay for retrying a flush after receiving an error.
 	defaultErrorDelay = 10 * time.Second
 	// defaultWorkerCount is a default number of workers that flush objects.
 	defaultWorkerCount = 20
+	// defaultMaxBatchDelay is a default delay before pushing small objects.
+	defaultMaxBatchDelay = 10 * time.Millisecond
```

Comment:
WC can have this delay be substantially bigger. Remember that FSTree has to balance PUT delay (batching obviously adds some time to every object) and throughput. WC can have this balance shifted significantly to throughput because it already holds the object and PUT was finished some time ago, now it's an internal node thing. A second is perfectly fine here.